### DISCUSSION TITLE
Vercel deploy error (next-images, styled-components)

### DISCUSSION THREAD

**Discussion by vagnermengali** [vercel/next.js#44908] [Help]
I'm trying to deploy but I'm not succeeding, 
I've already deleted the repositories and redid them again and it still gives the error

<html>
<body>

Cloning github.com/vagnermengali/rick-and-morty (Branch: main, Commit: 2f82087)
--
19:11:37.628 | Previous build cache not available
19:11:37.843 | Cloning completed: 468.338ms
19:11:37.933 | Running "vercel build"
19:11:38.401 | Vercel CLI 28.11.1
19:11:38.645 | Installing dependencies...
19:11:39.001 | yarn install v1.22.17
19:11:39.050 | [1/4] Resolving packages...
19:11:39.191 | [2/4] Fetching packages...
19:11:52.369 | [3/4] Linking dependencies...
19:11:52.372 | warning " > next-images@1.8.4" has unmet peer dependency "webpack@^4.0.0 \|\| ^5.0.0".
19:11:52.372 | warning "next-images > file-loader@6.2.0" has unmet peer dependency "webpack@^4.0.0 \|\| ^5.0.0".
19:11:52.372 | warning "next-images > url-loader@4.1.1" has unmet peer dependency "webpack@^4.0.0 \|\| ^5.0.0".
19:11:52.372 | warning " > styled-components@5.3.6" has unmet peer dependency "react-is@>= 16.8.0".
19:12:04.862 | [4/4] Building fresh packages...
19:12:04.988 | Done in 25.99s.
19:12:05.016 | Detected Next.js version: 13.1.2
19:12:05.017 | Running "yarn run build"
19:12:05.287 | yarn run v1.22.17
19:12:05.312 | $ next build
19:12:05.773 | Attention: Next.js now collects completely anonymous telemetry regarding usage.
19:12:05.773 | This information is used to shape Next.js' roadmap and prioritize features.
19:12:05.773 | You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
19:12:05.773 | https://nextjs.org/telemetry
19:12:05.773 |  
19:12:05.974 | info  - Linting and checking validity of types...
19:12:08.147 |  
19:12:08.147 | Failed to compile.
19:12:08.148 |  
19:12:08.148 | ./src/components/Header/Sidebar/index.tsx
19:12:08.148 | 20:11  Warning: Using `<img>` could result in slower LCP and higher bandwidth. Use `<Image />` from `next/image` instead to utilize Image Optimization. See: https://nextjs.org/docs/messages/no-img-element  @next/next/no-img-element
19:12:08.148 | 20:11  Warning: img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.  jsx-a11y/alt-text
19:12:08.148 |  
19:12:08.148 | ./src/components/Header/index.tsx
19:12:08.148 | 24:15  Warning: Using `<img>` could result in slower LCP and higher bandwidth. Use `<Image />` from `next/image` instead to utilize Image Optimization. See: https://nextjs.org/docs/messages/no-img-element  @next/next/no-img-element
19:12:08.148 |  
19:12:08.148 | ./src/components/Homepage/CollectionsCards/Cards/index.tsx
19:12:08.148 | 30:13  Error: Missing "key" prop for element in iterator  react/jsx-key
19:12:08.148 |  
19:12:08.149 | info  - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/basic-features/eslint#disabling-rules
19:12:08.187 | error Command failed with exit code 1.
19:12:08.188 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
19:12:08.226 | Error: Command "yarn run build" exited with 1

</body>
</html>

**Comment by gaston-flores** [comment]
You're getting an eslint error when building your app. See these lines in the build log:

>19:12:08.148 | ./src/components/Homepage/CollectionsCards/Cards/index.tsx
>19:12:08.148 | 30:13 Error: Missing "key" prop for element in iterator react/jsx-key

The mapped list [here](https://github.com/vagnermengali/rick-and-morty/blob/main/src/components/Homepage/CollectionsCards/Cards/index.tsx#L30) is missing a key prop. You can use `yarn build` in your local env to test that it works correctly once you've added it.
